src/client/components/AddMeal.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from "react";
+import Header from "./Header";
+import Footer from "./Footer";
+
+export default function AddMeal() {
+  const [inputState, setInputstate] = useState(false);
```

Comment:
Ah, I guess this is something like "isSending", but you never use it to disable the submit button.